docs/commit-messages.md

```diff
@@ -108,6 +108,51 @@ For changes in `tailscale/tailscale` that fix a significant bug or add a new fea
 add `RELNOTE: <summary of change>` toward the end of the commit message.
 This will aid the release engineer in writing the release notes for the next release.
 
+## Is it a #cleanup?
+
+Our issuebot permits writing `Updates #cleanup` instead of an actual GitHub issue number.
+
+But only do that if it’s actually a cleanup. Don’t use that as a shortcut[^1] to avoid filing a bug.
+
+[^1]: Shortcuts[^2] to file bugs: [go/bugc](http://go/bugc) (corp, safe choice) or [go/bugo](http://go/bugo) (open source, if you want it public to the world).
```

Comment:
I think this should probably still be inline, rather than a footnote.  I think the use of "shortcut" in this sentence and the previous are subtly different... this isn't actually explaining the meaning of "shortcut" in the previous sentence. (I know this language is from the original doc, and it's kinda confusing there too)

I do really like the footnote below though! ❤️

Reply:
Reworded to avoid ambiguity in [e4d822d..8b08892](https://github.com/tailscale/tailscale/compare/e4d822d1e916549590960308282cae0c51261e28..8b088924648b54f89f179024d43838be604ceeef).